lib/was/registrar/register_crawl_object_from_file.rb

```diff
@@ -0,0 +1,65 @@
+module Was
+  module Registrar
+    # Registers a set of crawl objects defined by an input file
+    class RegisterCrawlObjectFromFile
+      
+      # Registers the items in the input_file_path and log the errors in log_file
+      # @example file format:
+      #   Source_id job_directory collection_id
+      #   WAS:Test  jobs1/warc    druid:ab123cd4567
+      # @param input_file_path [String] the path to the crawl object list
+      # @param log_file [String] the path to the log file
+      def self.register(input_file_path, log_file)
+        return unless verify_file(input_file_path)
+
+        logger = Logger.new(log_file)
+        success_count = 0
+        fail_count = 0
+        registrar = Was::Registrar::RegisterCrawlObject.new
+
+        input_file = File.open(input_file_path)
```

Comment:
Consider closing the file within self.register: http://stackoverflow.com/questions/4795447/rubys-file-open-and-the-need-for-f-close